readme.md

```diff
@@ -1,3 +1,15 @@
 ## APO Laravel
 
-This is the source code for [apo.case.edu](http://www.apo.case.edu).
\ No newline at end of file
+This is the source code for [apo.case.edu](http://www.apo.case.edu).
+
+### Contributing
+
+The website is built on the Laravel framework for php. Development is
+is done in a vagrant box to maintain sanity accross development platforms.
+The box is known as Laravel Homestead. This also has the effect of making the
+initial setup a breeze!
+
+After cloning the project and starting a vagrant box,
+migrating the database is the last step you need before developing on
```

Comment:
I'm going to work on the next couple weeks  on setting up fake data generation for this so that we can allow non-apo people to work on this.